app/code/Magento/Config/Model/Config/PathValidator.php

```diff
@@ -40,6 +40,11 @@ public function __construct(Structure $structure)
      */
     public function validate($path)
     {
+        $element = $this->structure->getElementByConfigPath($path);
+        if ($element && method_exists($element, 'getConfigPath') && $element->getConfigPath()!==null) {
```

Comment:
1. Instead of checking that method exists, could we check here that the element is an instance of some class, or maybe not an instance of an element group?
https://github.com/magento/magento2/blob/fbfac3e51730abe2b79c6072544bc9742fcf7606/app/code/Magento/Config/Block/System/Config/Form.php#L301-L304

2. I believe if the `$element->getConfigPath()` will return empty string that's not also good option. I think we should just check `if(... && $element->getConfigPath()) {`. What do you think about it?

Reply:
Has been refactored.